QuizList/QuestionView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  QuestionView.swift
+//  QuizList
+//
+//  Created by Alexander v. Below on 26.08.20.
+//  Copyright © 2020 None. All rights reserved.
+//
+
+import SwiftUI
+
+extension Text {
+    public func highlight(_ highlight: Bool) -> some View {
+        if highlight {
+            return self.foregroundColor(.red)
+        } else {
+            return self
+        }
+    }
+}
+
+struct QuestionView: View {
+    var list: QuizList
+    var quizFactory: QuestionManufactory
```

Comment:
Looks like list and quizFactory can be made private.

Reply:
I always wonder why. Recently, I ran into problems because someone aggressively made things private …